.styleci.yml

```diff
@@ -3,13 +3,4 @@ preset: recommended
 linting: true
 
 finder:
-    exclude:
-        - "tests"
-    name:
-        - "*.php"
-    path:
-        - "src"
-    not-path:
-        - "tests"
-    depth:
-        - "< 3"
```

Comment:
Not sure either. Just copied the example and played with it to see how it works. I'll bring more changes to it while I'll read the documentation.

thanks for mention that.